sdk/storage/Azure.Storage.DataMovement.Blobs/src/AppendBlobStorageResource.cs

```diff
@@ -254,6 +256,30 @@ public override async Task<StorageResourceProperties> GetPropertiesAsync(Cancell
             return response.Value.ToStorageResourceProperties();
         }
 
+        /// <summary>
+        /// Gets the HTTP Authorization header for the storage resource if available.
```

Comment:
Nit: Docstring and return description could be updated for AccessToken now. This applies to all implementations of this method in all resources.